doc/news/_preparation_next_release.md

```diff
@@ -315,19 +315,19 @@ you up to date with the multi-language support provided by Elektra.
 - The build jobs now print less non-relevant output. *(René Schwaiger)*
 - Enable `-Werror` in `debian-stable-full`. *(Lukas Winkler)
 - We added the compiler switch `-Werror` to the build jobs:
```

Comment:
Wasn't it added to all the jobs in the end? Maybe also mention where it is not done yet, if you already know the list.

And you forgot to mention your most important contribution: you did not only add "-Werror" but you also fixed all warnings so that it actually works :heart:

Reply:
> Wasn't it added to all the jobs in the end?

Only the job for Debian “Jessie“ is missing. However, compiling Elektra does not work anymore if we add the compiler switch `-Werror` on that OS. The reason behind that is that `find_package (Threads)` fails, if we use CMake `3.0.2` and add said switch.

Reply:
That is perfectly fine if it does not work for this legacy system, I only wanted to say it would be nice if it is documented in the release notes.